mllib/src/main/scala/org/apache/spark/mllib/optimization/GradientDescent.scala

```diff
@@ -142,7 +157,10 @@ object GradientDescent extends Logging {
    * @param regParam - regularization parameter
    * @param miniBatchFraction - fraction of the input data set that should be used for
    *                            one iteration of SGD. Default value 1.0.
-   *
+   * @param convergenceTol - Minibatch iteration will end before numIterations if the
```

Comment:
There should not be a leading "-" for these parameters.  (I know you're just following previous work, but can you please remove them?)

Also, please update doc based on using weight vector (not loss) and using relative change